<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JalousieEinstellungen</class>
 <widget class="QDialog" name="JalousieEinstellungen">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>571</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Jalousiesteuerung - Einstellungen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_JalReset">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>460</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rücksetzen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_JalOK">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>460</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Übernehmen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_JalCancel">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>460</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Abbrechen</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_Zeitautomatik">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>691</width>
     <height>261</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QCheckBox" name="checkBox_Zeitautomatik">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Zeitautomatik</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_Zu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>671</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>651</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Schließen:</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QRadioButton" name="radioButton_Zu_Uhrzeit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>28</y>
        <width>109</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Uhrzeit</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_Zu_SA">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>28</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sonnenaufgang</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_Zu_SU">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>55</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sonnenuntergang</string>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>26</y>
        <width>3</width>
        <height>52</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_Zu_Offset">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>49</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QToolButton" name="toolButton_9">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>0</y>
         <width>30</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="hsGui.qrc">
         <normaloff>:/new/jalousie/resource/Ab.png</normaloff>:/new/jalousie/resource/Ab.png</iconset>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>1</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Offset:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>spinBox_Zu_Offset</cstring>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_Zu_Offset">
       <property name="geometry">
        <rect>
         <x>68</x>
         <y>0</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> min</string>
       </property>
       <property name="minimum">
        <number>-180</number>
       </property>
       <property name="maximum">
        <number>180</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_10">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>0</y>
         <width>30</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="hsGui.qrc">
         <normaloff>:/new/jalousie/resource/Auf.png</normaloff>:/new/jalousie/resource/Auf.png</iconset>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_Zu_Uhrzeit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QToolButton" name="toolButton_12">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>0</y>
         <width>30</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="hsGui.qrc">
         <normaloff>:/new/jalousie/resource/Ab.png</normaloff>:/new/jalousie/resource/Ab.png</iconset>
       </property>
      </widget>
      <widget class="QTimeEdit" name="timeEdit_Zu">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>0</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_11">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>0</y>
         <width>30</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="hsGui.qrc">
         <normaloff>:/new/jalousie/resource/Auf.png</normaloff>:/new/jalousie/resource/Auf.png</iconset>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_Auf">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>671</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>651</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Öffnen:</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QRadioButton" name="radioButton_Auf_Uhrzeit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>28</y>
        <width>109</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Uhrzeit</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_Auf_SA">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>28</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sonnenaufgang</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_Auf_SU">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>55</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sonnenuntergang</string>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>26</y>
        <width>3</width>
        <height>52</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_Auf_Offset">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>49</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QToolButton" name="toolButton_13">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>0</y>
         <width>30</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="hsGui.qrc">
         <normaloff>:/new/jalousie/resource/Ab.png</normaloff>:/new/jalousie/resource/Ab.png</iconset>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>1</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Offset:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>spinBox_Auf_Offset</cstring>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_Auf_Offset">
       <property name="geometry">
        <rect>
         <x>68</x>
         <y>0</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string> min</string>
       </property>
       <property name="minimum">
        <number>-180</number>
       </property>
       <property name="maximum">
        <number>180</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_14">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>0</y>
         <width>30</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="hsGui.qrc">
         <normaloff>:/new/jalousie/resource/Auf.png</normaloff>:/new/jalousie/resource/Auf.png</iconset>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_Auf_Uhrzeit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QToolButton" name="toolButton_15">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>0</y>
         <width>30</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="hsGui.qrc">
         <normaloff>:/new/jalousie/resource/Ab.png</normaloff>:/new/jalousie/resource/Ab.png</iconset>
       </property>
      </widget>
      <widget class="QTimeEdit" name="timeEdit_Auf">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>0</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_16">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>0</y>
         <width>30</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="hsGui.qrc">
         <normaloff>:/new/jalousie/resource/Auf.png</normaloff>:/new/jalousie/resource/Auf.png</iconset>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_Optionen">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>170</y>
     <width>201</width>
     <height>261</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Optionen:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_aufLuecke">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Auf Lücke</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_TuerOffen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>182</y>
      <width>133</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tür bleibt offen</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Wettersteuerung">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>61</y>
      <width>155</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wettersteuerung</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_OpenRain">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>119</y>
      <width>155</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Öffnen bei Regen</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_OpenWind">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>155</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Öffnen bei Wind</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="hsGui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_JalCancel</sender>
   <signal>clicked()</signal>
   <receiver>JalousieEinstellungen</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>892</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>929</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_10</sender>
   <signal>clicked()</signal>
   <receiver>spinBox_Zu_Offset</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>660</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>641</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_9</sender>
   <signal>clicked()</signal>
   <receiver>spinBox_Zu_Offset</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>693</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>632</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_14</sender>
   <signal>clicked()</signal>
   <receiver>spinBox_Auf_Offset</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>662</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>640</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_13</sender>
   <signal>clicked()</signal>
   <receiver>spinBox_Auf_Offset</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>695</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>633</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_16</sender>
   <signal>clicked()</signal>
   <receiver>timeEdit_Auf</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_15</sender>
   <signal>clicked()</signal>
   <receiver>timeEdit_Auf</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_11</sender>
   <signal>clicked()</signal>
   <receiver>timeEdit_Zu</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_12</sender>
   <signal>clicked()</signal>
   <receiver>timeEdit_Zu</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
